packages/project-management-automation/lib/tasks/first-time-contributor-account-link/index.js

```diff
@@ -86,9 +86,11 @@ async function firstTimeContributorAccountLink( payload, octokit ) {
 	try {
 		hasProfile = await hasWordPressProfile( author );
 	} catch ( error ) {
-		debug(
-			`first-time-contributor-account-link: Error retrieving from profile API:\n\n${ error.toString() }`
-		);
+		if ( error instanceof Error ) {
```

Comment:
The question is whether `request` throws an `Error` instance.

Reply:
It may be better to change the original promise so that it [creates a new error when rejecting](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/reject#description):

`reject(new Error('fail'))`

Reply:
`error.toString()` would print `fail` in that case.

I came up with a simpler solution. I added a check if `error` is an object which should be general enough and ensure there is `toString` method present 😄